react-native-sdk/package.json

```diff
@@ -84,15 +85,14 @@
         "react-native-video": "0.0.0",
         "react-native-watch-connectivity": "0.0.0",
         "react-native-webrtc": "0.0.0",
-        "react-native-webview": "0.0.0",
-        "text-encoding": "0.0.0"
+        "react-native-webview": "0.0.0"
     },
     "overrides": {
         "@xmldom/xmldom": "0.0.0"
     },
     "scripts": {
         "postinstall": "node sdk_instructions.js",
-        "prepare": "node prepare_sdk.js"
+        "prepare-sdk-deps": "node update_sdk_dependencies.js"
```

Comment:
Undo this. Prepare is a standard step before publish and we need it to copy the native files.

Dependencies are updated with the shell script that then calls into node.